app/models/storage_tables/attachment.rb

```diff
@@ -19,6 +19,10 @@ def path
       association(:blob).klass.service.path_for(full_checksum)
     end
 
+    def relative_path
+      association(:blob).klass.service.relative_path_for(full_checksum)
```

Comment:
Ik denk dat het verstandig is hier (en bij path en download) nil te returnen als er geen checksum is. Of wellicht een exception raisen. Je krijgt anders onzin terug waar ActiveStorage wel een beveiliging voor heeft.

```ruby
ActiveStorage::Attachment.new.path
=> Module::DelegationError: path delegated to blob, but blob is nil

StorageTables::InvoiceDocument.new.path
=> "/home/nicke/apps/debiteurenbeheer/storage/=/=/=="
```

Reply:
Raisen is hier het beste, anders kan het zijn dat je een nil doorgeeft aan een gebruiker, terwijl dit eigenlijk een error is.